app/components/PurchaseTickets/Form.js

```diff
@@ -216,10 +217,18 @@ const PurchaseTicketsForm = ({
         </div>
       )}
 
-      <KeyBlueButton
-        className="stakepool-content-import-script-button"
-        onClick={onShowImportScript}
-      >Import Script</KeyBlueButton>
+      <div className="stakepool-content-import-script-button"
+        data-html={true}
+        data-tip-disable={rescanRequest ? false : true}
```

Comment:
Shouldn't it be disabled if `rescanRequest` is `true`? Also, it can be simplified to `data-tip-disable={rescanRequest}`.

Reply:
The tip is to be shown while the rescan is running. Though just the tip there is probably not ideal (maybe a "?" icon so the user knows something is up during the rescan).

Reply:
Though here the test *can* be simplified to just {rescanRequest} (no type check on data-tip-disabled)

Reply:
Ah sorry, I should have looked a little closer.